{"uuid":"73cea0db-34cf-4815-a437-b3c05679f8de","historyId":"4b8287d8d696f88bd133f09643ced2fc","fullName":"com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword","labels":[{"name":"package","value":"com.leadgain.testcase.LoginTest"},{"name":"testClass","value":"com.leadgain.testcase.LoginTest"},{"name":"testMethod","value":"invalidLoginTest_InvalidUserNameInvalidPassword"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"com.leadgain.testcase.LoginTest"},{"name":"host","value":"kashmivm"},{"name":"thread","value":"3292@kashmivm.main(1)"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"Login Tests"},{"name":"story","value":"Invalid username and password login test"},{"name":"severity","value":"blocker"}],"links":[],"name":"Invalid Login Scenario with wrong username and password.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Actual heading '' should be same as expected 'LEADGAIN'. expected [LEADGAIN] but found []","trace":"java.lang.AssertionError: Actual heading '' should be same as expected 'LEADGAIN'. expected [LEADGAIN] but found []\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:453)\r\n\tat com.leadgain.page.LoginPage.verifyLoginPageHeader(LoginPage.java:69)\r\n\tat com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTest.java:52)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:230)\r\n\tat org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:242)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:579)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)\r\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\r\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)\r\n"},"stage":"finished","description":"Test Description: Login test with wrong username and wrong password.","start":1539056806651,"stop":1539056814112,"steps":[{"name":"Open url ...","status":"passed","stage":"finished","start":1539056806682,"stop":1539056813146,"parameters":[{"name":"url","value":"https://www.demandmatrix.com/leadgaindev/"}]}],"attachments":[{"name":"Page screenshot","source":"08969b05-36dc-4746-9344-2fbba723aba1-attachment","type":"image/png"},{"name":"{0}","source":"8297fe8e-5755-48a8-9cc5-5b03c1b9fff0-attachment","type":"text/plain"}],"parameters":[{"name":"method","value":"public void com.leadgain.testcase.LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method) throws java.lang.InterruptedException"}]}